playbooks/test.yml

```diff
@@ -15,7 +15,7 @@
         broker_dao_type: crd
 
     - name: Wait for clusterservicebroker to become available
-      command: kubectl get clusterservicebrokers automation-broker -o go-template={%raw%}'{{ range .status.conditions }}{{ if eq .type "Ready" }}{{ .status }}{{ end }}{{ end }}'{%endraw%}
+      command: kubectl get clusterservicebrokers ansible-service-broker -o go-template={%raw%}'{{ range .status.conditions }}{{ if eq .type "Ready" }}{{ .status }}{{ end }}{{ end }}'{%endraw%}
```

Comment:
`ansible-service-broker` should just be `{{ broker_name }}`

Reply:
If you change the broker name it will install into the namespace ansible-service-broker. We have a mandate from the --enable option that it should be prefixed with openshift- so that won't work.

Even if you want the broker name to be ansible-service-broker inside openshift-ansible-service-broker I now have to update the --enable code since I removed the broker_namespace option.

Not to mention we should probably set the CRD name to ansible-service-broker regardless of what they set the broker name to because changing it from ansible-service-broker breaks apb tool.

We talked about this Friday. I thought you and @dymurray agreed to defaulting to ansible-service-broker for the CRD name.

Reply:
I'm in favor of defaulting to `ansible-service-broker`. I thought @djzager's comment was just that we should use the variable so that it can be reused even if we do want to change it.